docs/content/labs/common/hypervisor/requisites.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Hypervisor Prerequisites
+---
+
+- **CPU**: The number of CPUs provided determines how many HostedClusters can run concurrently.
+  - **Recommended**: 16 CPUs per Node for 3 nodes.
+  - **Minimal Dev**: In a development environment, you may manage with 12 CPUs per Node for 3 nodes.
+
+- **Memory**: The amount of RAM impacts how many HostedClusters can be hosted.
+  - **Recommended**: 48 GB of RAM per Node.
+  - **Minimal Dev**: For minimal development, 18 GB of RAM per Node may suffice.
+
+- **Storage**: Using SSD storage for MCE is crucial.
```

Comment:
I don't think ssd is enough, I think it needs to be NVME, for etcd PVs, right?

Reply:
for a development environment I think SSD is enough, our BM env works fine under SSD, but if you want to run more than 1 serious hosted cluster, yes NVME is mandatory.